plugins/modules/net_tools/ldap/ldap_entry.py

```diff
@@ -131,6 +138,8 @@ def __init__(self, module):
         # Load attributes
         if self.state == 'present':
             self.attrs = self._load_attrs()
+        else:
+            self.recursive = self.module.params['recursive']
```

Comment:
This isn't related to attributes.
```suggestion

        # Flag for recursive deletion
        self.recursive = self.module.params['recursive']
```

Reply:
You're right. Rather than doing that, I will move the line right under `self.state = ...` as it is a shortcut as well and the code is so obvious it doesn't really deserve its own comment.

Reply:
Done in commit e6f2a8b6